scripts/readelf.py

```diff
@@ -425,6 +426,10 @@ def display_dynamic_tags(self):
                     parsed = 'Library soname: [%s]' % tag.soname
                 elif tag.entry.d_tag.endswith(('SZ', 'ENT')):
                     parsed = '%i (bytes)' % tag['d_val']
+                elif tag.entry.d_tag == 'DT_FLAGS':
+                    parsed = describe_dt_flags(tag.entry.d_val)
```

Comment:
No "Flags: " needed here?

Reply:
That's what my system's `readelf` prints, so I figured I'd add it here as well.

Reply:
Ah, sorry, I just realized I read your comment wrong. For DT_FLAGS, `readelf` only shows the respective flag names, while for DT_FLAGS_1, the additional "Flags: " is printed... This is also the case in the `readelf` binary shipped with the tests.